lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.spec.ts

```diff
@@ -1304,5 +1304,24 @@ describe('ContentNodeSelectorPanelComponent', () => {
                 expect(component.sorting).toEqual(['createdAt', 'desc']);
             });
         });
+
+        describe('Selected nodes counter', () => {
```

Comment:
Could you add an extra check that checks the binding with the html? Right now your test is checking the getselectedCount() method but if someone changes the html that you added, the counter could be broken without any tests highlighting a problem